### PR TITLE
Add filter parameter to find user

### PR DESCRIPTION
**Pull request by djino97**
https://github.com/LT-Students/DigitalOffice-UserService/pull/55/files

### CODE REVIEW

File: src/UserService.Models.Dto/Configurations/RabbitMqConfig.cs [hunk 1]
```diff
@@ -1,24 +1,58 @@
...
+using LT.DigitalOffice.Models.Broker.Requests.Company;
+using LT.DigitalOffice.Models.Broker.Requests.File;
+using LT.DigitalOffice.Models.Broker.Requests.Message;
+using LT.DigitalOffice.Models.Broker.Requests.Project;
+using LT.DigitalOffice.Models.Broker.Requests.Token;
+using LT.DigitalOffice.Models.Broker.Requests.User;
 
 namespace LT.DigitalOffice.UserService.Models.Dto.Configurations
 {
     public class RabbitMqConfig : BaseRabbitMqConfig
```
Comment (BelehovEgor) [inline src/UserService.Models.Dto/Configurations/RabbitMqConfig.cs:12]: here i guess you have extra injects, check which requests used in consumers and don't inject them

**Review by BelehovEgor — CHANGES_REQUESTED** [review]

**Review by KostinP — APPROVED** [review]

File: src/UserService.Models.Dto/Configurations/RabbitMqConfig.cs (hunk 1 above)
Comment (djino97) [reply src/UserService.Models.Dto/Configurations/RabbitMqConfig.cs:11]: Fixed

File: src/UserService.Models.Dto/Configurations/RabbitMqConfig.cs [hunk 2]
```diff
@@ -1,24 +1,55 @@
...
+using LT.DigitalOffice.Models.Broker.Requests.Message;
+using LT.DigitalOffice.Models.Broker.Requests.Project;
+using LT.DigitalOffice.Models.Broker.Requests.Token;
 
 namespace LT.DigitalOffice.UserService.Models.Dto.Configurations
 {
     public class RabbitMqConfig : BaseRabbitMqConfig
     {
         public string CompanyServiceUrl { get; set; }
+
```
Comment (BelehovEgor) [inline src/UserService.Models.Dto/Configurations/RabbitMqConfig.cs:14]: extra empty line

**Review by BelehovEgor — APPROVED** [review]

File: src/UserService.Models.Dto/Configurations/RabbitMqConfig.cs (hunk 2 above)
Comment (djino97) [reply src/UserService.Models.Dto/Configurations/RabbitMqConfig.cs:14]: no, this need separated

**Review by MariyaSin — APPROVED** [review]

**Review by TIove — APPROVED** [review]

**Pull request merged**